website/docs/api/Protocols.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: protocols
-title: Protocol Commands
+title: Communication Protocols
 ---
 
-WebdriverIO is an automation framework that relies on various of automation protocols to control a remote agent, e.g. a browser, mobile device or television. Based on the remote device different protocols come into play. These commands are assigned to the [Browser](/docs/api/browser) or [Element](/docs/api/element) Object depending on the session information by the remote server (e.g. browser driver).
+WebdriverIO (WDIO) is an automation framework that relies on various internal protocols to control a remote agent, for e.g. a browser, mobile device or television. Based on the remote device different protocols come into play. These commands are assigned to the [Browser](/docs/api/browser) or [Element](/docs/api/element) Object depending on the session information by the remote agent's server (e.g. browser driver) automatically.
```

Comment:
> various internal protocols

I don't think these protocols are internal. The WebDriver protocol is a web standard so not internal at all.

Reply:
Internal in the sense under the hood. Testing developers the main audience of the documentation would not need to have any idea of Webdriver or other protocols that WDIO uses. WDIO is handling it from them. They just need to install WDIO, config it, use WDIO methods to get the tests done imo